pandas/plotting/_converter.py

```diff
@@ -178,6 +180,16 @@ class DatetimeConverter(dates.DateConverter):
 
     @staticmethod
     def convert(values, unit, axis):
+        # values might be a 1-d array, or a list-like of arrays.
+        if is_list_like(values) and is_list_like(values[0]):
```

Comment:
use

``is_list_like(values) and all(is_list_like(v) for v in values)``.

you could make this a method in ``pandas.core.dtypes.common`` if you want (with tests!) maybe
``is_list_like_nested`` (I don't know if there is a case where ``any`` of the sub-lists is True but you don't want ``all``)